dev-python/gevent/Manifest

```diff
@@ -3,3 +3,4 @@ DIST gevent-20.5.1.tar.gz 5566653 BLAKE2B 1d9a474681573ec2e9d3c0d40fbae8a91b8cfb
 DIST gevent-20.5.2.tar.gz 5567755 BLAKE2B 9cac45f92c7c1841cca4eb52a54f71406beb3e83136170610bba326a32722de3c5ebd3b777a45b73ab5e1012eb0dc49d861b042bab0c6194e61f44e6011ede2e SHA512 24f00a599f26d673947f0ed5582f8b55d1b214677578d7c5c2f8d8c106f89a98951db523597c470bcd788de9f6b7abcd3f25f7aff27788227f6d8c7ca7c183c1
 DIST gevent-20.6.0.tar.gz 5803237 BLAKE2B 97b988964ef80fcc1c3238718b6802016fb46742c94e7e56b5196139181ff82ce60816e0a8ccc5c3e1c3b73ec11cc705884b5ac5524dbd5116dc0d8722ccc7a4 SHA512 28ee030934d3c379577c9f86b76838e5d76c7ea7c0880cdeac9dc5ead497ae0600fb02b2984c7312fd42319142200c674b75da2e93b58fd4a99882d601a95108
 DIST gevent-20.6.1.tar.gz 5806150 BLAKE2B fb75e440f31ad2952d1395780a66ee673c55e62b635a2c6757efe9706c404d126c38a613ac08cd088900c0b10913fe9ce9d57cec8a7cb252ae87912142cfafce SHA512 b7ceb5251b01972a11e3d51e1ebcaaca7c08d36b6ba7b138bee57f0d5da4553539a0ad538e8a433321bdac396788acbfb0ac294191e829df3f066d00816310bb
+DIST gevent-20.6.2.tar.gz 5806612 BLAKE2B 8b1fbd991cb169292d33001bb0df06702d36eb3849c9251e4fac9e2fb5826eda3435971e39e209f1f537536081ce2730e561e91c398ee66a9b1d55c9d0e5a81a SHA512 65caa9caf8a2913b4c3bcc9a303169ce5837a0d7ccf4fb6b7583bf4ec3a0b288d65fc7e28399f0715e61aed128ffc6e67fc9b455474e47263236136a492021f4
```

Comment:
This doesn't seem to belong to this commit.

Reply:
Oops yes, that should have been with the version bump commit, will fix and rebase.